### PR TITLE
Navigation bar and category page

### PR DESCRIPTION
**Pull request by nickhsine**
@hcchien

### CODE REVIEW

File: src/components/Tags.js [hunk 1]
```diff
@@ -19,15 +35,11 @@ export default class Tags extends Component {
     if (articles && articles.length > 0) {
       return (
         <div className="category-items">
-          <div className="category-name">
-            <Category>{cat_display}</Category>
-          </div>
+        <cat cat_display={cat_display}/>
```
Comment (nickhsine) [inline src/components/Tags.js]: @hcchien where is ```cat``` ?

Comment (hcchien) [inline src/components/Tags.js]: it still doesn't work. so I just remove it first.

**Comment by hcchien** [comment]
we will need to wait for the design spec before merge.

**Comment by hcchien** [comment]
ok, merge it.

**Pull request merged**